api/management/commands/acquiredata.py

```diff
@@ -27,3 +27,13 @@ def handle(self, *args, **options):
         if not os.path.exists(sample_path):
             sample_url = 'https://raw.githubusercontent.com/cognoma/cancer-data/master/data/samples.tsv'
             urlretrieve(sample_url, sample_path)
+
+        gene_path = os.path.join(options['path'], 'genes.tsv')
+        if not os.path.exists(gene_path):
+            gene_url = 'http://www.stephenshank.com/genes.tsv'
+            urlretrieve(gene_url, gene_path)
+
+        mutation_path = os.path.join(options['path'], 'mutation-matrix.tsv.bz2')
+        if not os.path.exists(mutation_path):
+            mutation_url = 'https://ndownloader.figshare.com/files/5864862'
```

Comment:
Would be nice if we could use the figshare download logic from `cognoml`. See [`cognoml/figshare.py`](https://github.com/cognoma/cognoml/blob/33045e853e57e18c06f64f6ab499da3081080a5a/cognoml/figshare.py). @jessept is our code modular enough that @stephenshank can use cognoml to download the figshare data here, or would this application be out of scope.

I created a corresponding issue for the `cognoml` team: https://github.com/cognoma/cognoml/issues/15.